src/DetailsView/components/issues-table.tsx

```diff
@@ -183,11 +183,21 @@ export class IssuesTable extends React.Component<IssuesTableProps> {
         const isCardsUIEnabled = this.props.featureFlags[FeatureFlags.universalCardsUI];
         const commandBar = !isCardsUIEnabled ? this.renderCommandBar() : null;
 
+        const disabledMessage = isCardsUIEnabled ? (
+            <span>
+                Use the <Markup.Term>start over</Markup.Term> button to scan the target page.
```

Comment:
Should this be "Start over" for consistency with the button text?

Reply:
(with capital S)

Reply:
updated